libs/wire-subsystems/src/Wire/InvitationStore.hs

```diff
@@ -18,25 +18,21 @@
 {-# LANGUAGE StrictData #-}
 {-# LANGUAGE TemplateHaskell #-}
 
-module Wire.InvitationCodeStore where
+module Wire.InvitationStore where
```

Comment:
:+1:

Reply:
The store is not about codes, hence the name shouldn't have "code" in it.